BDArmory/Weapons/Missiles/MissileBase.cs

```diff
@@ -1091,7 +1091,7 @@ protected void UpdateRadarTarget()
                 {
                     radarTarget = TargetSignatureData.noTarget;
                     TargetAcquired = true;
-                    TargetPosition = transform.position + (startDirection * 500);
+                    TargetPosition = transform.position + (startDirection * 5000);
                     TargetVelocity = Vector3.zero;
```

Comment:
If we also set
```csharp
TargetVelocity = vessel.Velocity(); // Set the relative target velocity to 0.
```
Then we can use
```csharp
if (Vector3.Dot(missile.TargetVelocity - missile.vessel.Velocity(), missile.GetForwardTransform()) >= 0) continue; // Ignore missiles that aren't gaining on their targets.
```
as an extra check in `UpdateCamera` for which missile to switch to.

Would this cause any problems with the missile guidance?